.gitignore

```diff
@@ -109,4 +109,7 @@ ENV/
 .DS_Store
 
 # Ignore recordings to keep tests local
-recordings
\ No newline at end of file
+recordings
+
+# Ignore meshDeploy folder
```

Comment:
Is this generated from testing?

Reply:
The deployment create command merges the yamls and generates combined jsons. These jsons are dumped in this directory and used for the deploying the resources.